invenio_iiif/__init__.py

```diff
@@ -6,7 +6,141 @@
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-"""IIIF API for Invenio."""
+"""Invenio module to serve images complying with IIIF standard.
+
+Invenio-IIIF integrates
+`Invenio-Records-Files <https://invenio-records-files.rtfd.io>`_ with
+`Flask-IIIF <https://flask-iiif.rtfd.io>`_ to provide an endpoint for serving
+images complying with the `International Image Interoperability Framework
+(IIIF) <https://iiif.io/>`_ API standards.
+
+Invenio-IIIF registers the REST API endpoint provided by Flask-IIIF in the
+Invenio instance through entry points. On each request, it delegates
+authorization check to
+`Invenio-Files-REST <https://invenio-files-rest.rtfd.io>`_, load the file from
+the Invenio bucket and serve it after Flask-IIIF manipulation.
+
+Initialization
+--------------
+
+First create a Flask application (Flask-CLI is not needed for Flask
```

Comment:
Can we remove `Flask-CLI is not needed for Flask version 1.0+)` since we do not initialize any CLI?